site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+                        Explore
+                      </button>
+                    </h4>
+                    <div id="exploreCollapseIconsOc" class="accordion-collapse collapse" aria-labelledby="exploreHeaderIconsOc" data-bs-parent="#dashboardAccordionIconsOc">
+                      <div class="accordion-body">
+                        <ul class="list-unstyled m-0">
+                          <li><a href="#" class="side-nav-item">Label</a></li>
+                          <li><a href="#" class="side-nav-item">Label</a></li>
+                          <li><a href="#" class="side-nav-item">Label</a></li>
+                        </ul>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="accordion-item">
+            <h3 class="accordion-header" id="chartsHeaderIconsOc">
+              <button class="accordion-button collapsed side-nav-item fw-bold" type="button" data-bs-toggle="collapse" data-bs-target="#chartsCollapseIconsOc" aria-expanded="false" aria-controls="chartsCollapseIconsOc">
+                <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#chart"/>
+                </svg>
+                Charts
+              </button>
+            </h3>
+            <div id="chartsCollapseIconsOc" class="accordion-collapse collapse" aria-labelledby="chartsHeaderIconsOc" data-bs-parent="#sideNavAccordionIconsOc">
+              <div class="accordion-body">
+                <ul class="list-unstyled m-0">
+                  <li><a href="#" class="side-nav-item">Label</a></li>
+                  <li><a href="#" class="side-nav-item">Label</a></li>
+                  <li><a href="#" class="side-nav-item">Label</a></li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <div class="accordion-item">
+            <a href="#" class="side-nav-item fw-bold">
+              <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#financial"/>
+              </svg>
+              Financial
+            </a>
+          </div>
+        </div>
+      </div>
+    </nav>
+
+  </div>
+</div>
+
+<div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+{{< /example >}}
+
+### Collapsible
+
+Here is the `.side-nav-collapsible` theme. The collapsible side navigation we provide uses [Tooltips]({{< docsref "/components/tooltips#usage" >}}) internally. It also uses a small trick to nicely collapse the side navigation with an empty `div`. (might change if we try to change the collapse thing on Bootstrap)
```

Comment:
I would not say that it may change. Or I would at least rephrase it.

Reply:
Ah yeah it was a note for myself, sorry for that 🤣 Please let this one open to remember about it !